src/views/ShipmentDetails.vue

```diff
@@ -129,6 +129,10 @@ export default defineComponent({
         .create({
           component: AddProductModal
         })
+        modal.onDidDismiss()
```

Comment:
Unwanted change. Is it intentional?

Reply:
Yes Sir, When we dismiss the modal by clicking on the backdrop, the products are not removed from modal and they are displayed when we open modal of any other shipment or purchase order.